### PR TITLE
fix building examples

### PR DESCRIPTION
**Pull request by rjeczalik**
Hey,

Running `go build ./...` fails with:

```
$ go build ./...
go build github.com/koding/kite/test: no buildable Go source files in /Users/rjeczalik/src/github.com/koding/kite/test
# github.com/koding/kite/examples/exp2-watch
examples/exp2-watch/exp2-watch.go:23: undefined: kite.Event
examples/exp2-watch/exp2-watch.go:28: k.WatchKites undefined (type *kite.Kite has no field or method WatchKites)
# github.com/koding/kite/examples/math
examples/math/math.go:27: r.Response undefined (type *kite.Request has no field or method Response)
```

This CL tries to fix that problem. I took opportunity and extended the math example to be a bit more interactive.

### CODE REVIEW

**Comment by fatih** [comment]
Hi @rjeczalik.

Thanks for the PR

Etcd is [not supported](https://github.com/gyuho/etcd/commit/6264a41e227a884d7ad7513d1cd0025f06febcc5)  anymore with Go 1.3. I think we should not support it either. @rjeczalik can you please remove it from wercker.yml? @cihangir Any objections for it?

**Comment by rjeczalik** [comment]
@fatih Do you want to keep testing go1.3 with postgres or removing it altogether is ok?

**Comment by fatih** [comment]
I want to remove it altogether. People now use mostly 1.4.x and 1.5.x Also I think we shouldn't support that much version with our current man power :)

**Comment by rjeczalik** [comment]
@fatih Ack, then done. Please take a look :)

**Comment by cihangir** [comment]
https://github.com/koding/kite/pull/138#issuecomment-145274308

:+1:

File: examples/math/config.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+func init() {
+	flag.Var(Host, "addr", "Network address of the kite server.")
+}
+
+type Config struct {
+	URL *url.URL
+}
+
+func (cfg *Config) IP() string {
```
Comment (fatih) [inline examples/math/config.go]: We usually one char names for receiver. So `cfg` -> `c`.

File: examples/math/config.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package math
```
Comment (fatih) [inline examples/math/config.go]: This is nice @rjeczalik. A really different way of exposing configurations. We for example use https://github.com/koding/multiconfig with a pre defined struct. It's more easier and maintainable in the long term. For this PR you don't need to update it as this is just an example :)

Comment (rjeczalik) [inline examples/math/config.go]: Yup, I've seen multiconfig, just thought external dependency may be too much for an example.

**Comment by rjeczalik** [comment]
@fatih Renamed the receivers, PTAL.

**Pull request merged**